web/src/components/vendor-users/VendorUserCard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-skeleton-loader
+    v-if="isNil(vendorUser)"
+    type="card"
+  />
```

Comment:
Interesting note about this pattern, while I find it super convenient, we've been getting some pushback on it in the WRAP project since it make the UI jumpy. I don't have a solution for that yet, but putting in on your radar as food for thought. If you have an alternate idea, please let me know.